reconstructSPI/iterative_refinement/expectation_maximization.py

```diff
@@ -0,0 +1,524 @@
+"""Iterative refinement with Bayesian expectation maximization."""
```

Comment:
I'm not sure about the name of this file and folder... maybe @ninamiolane and @fredericpoitevin have some suggestions.

I forsee some common methods being used for ab initio and iterative refinement, such as taking slices, computing bayesian weights on each slice. Should we refactor now, or later when there is another method?